onnxruntime/core/framework/prepacked_weights_container.cc

```diff
@@ -49,4 +61,87 @@ size_t PrepackedWeightsContainer::GetNumberOfElements() const {
   return prepacked_weights_map_.size();
 }
 
+PrepackedForSerialization::PrepackedForSerialization()
+    : main_graph_(nullptr, key_to_blobs_, false) {
+}
+
+PrepackedForSerialization::~PrepackedForSerialization() = default;
+
+void PrepackedForSerialization::Subgraph::InsertFromDisk(const std::string& key, PrePackedWeights&& packed_weight) {
```

Comment:
nit: the 'FromDisk' seems like it's a usage of the API but not an aspect of the implementation. i.e. we're not actually reading from disk here, so the implementation doesn't actually care where `packed_weight` came from.